src/views/packageRootNode.ts

```diff
@@ -20,6 +20,16 @@ export class PackageRootNode extends DataNode {
         super(nodeData, parent);
     }
 
+    public getLabel(): string {
+        if (this.getParent() instanceof ContainerNode) {
+            const parent = this.getParent() as ContainerNode;
+            if (parent.getContainerType() == ContainerType.Maven) {
```

Comment:
@jdneo , I've noticed that PackageRootNode is actually type of Java Package that Label needs to change, would this be feasable? I can check if it is a Maven type by looking at parent if it is ContainerNode, or other approach would be just to check for metaData.['maven.groupId']. What is more favourable approach?

Reply:
So far just check `just to check for metaData.['maven.groupId']` looks good to me.

Reply:
Done.